src/coreclr/tools/ILVerify/README.md

```diff
@@ -40,11 +40,41 @@ The code is split into three projects:
 
 ## Tests
 
-To test the ILVerification library we have small methods checked in as .il files testing specific verification scenarios. These tests live under [src/ILVerification/tests/ILTests](../ILVerification/tests/ILTests). Tests are grouped into .il files based on functionalities they test. There is no strict policy here, the goal is to have a few dozen .il files instead of thousands containing each only a single method.
+To test the ILVerification library we have small methods checked in as .il files testing specific verification scenarios. These tests live under [src/tests/ilverify/ILTests](../../../tests/ilverify/ILTests). Tests are grouped into .il files based on functionalities they test. There is no strict policy here, the goal is to have a few dozen .il files instead of thousands containing each only a single method.
 
 The test project itself is under [src/tests/ilverify](../../../tests/ilverify)
 
- Method names in the .il files must follow the following naming convention:
+### Building and Running Tests
+
+As the test project is marked with priority=1, simply building the test projects from the root of the project is not enough. For the initial build of priority=1 in release mode one must run the following:
+
+```sh
+src/tests/build.(cmd/sh) release -priority=1
+```
+
+It is important that one does not attempt to build the test project using `dotnet build` or `dotnet test`, as this will invalidate the state of the build and requires a full rebuild of both (see this [issue](https://github.com/dotnet/runtime/issues/43967)). In case one does break their build, one must clean the build artifacts and rebuild in order to build the tests correctly. The following git command can be used to clean the repository from untracked changes, such as the build artifacts.
+
+```sh
+git clean -xdf
+```
+
```

Comment:
```suggestion
It is important that one does not attempt to build the test project using `dotnet build` or `dotnet test`, as this will invalidate the state of the build and requires a full rebuild of both (see this [issue](https://github.com/dotnet/runtime/issues/43967)).
```
The additional details on how to clean your enlistment break the flow of the document. They can be found in the linked issue if necessary.